templates/readme-template.md

```diff
@@ -0,0 +1,50 @@
+# Module Title
+
+## Description
+<!-- provide a summary of the purpose and use-case for your module -->
+
+## Requirements
+<!-- 
+list any kind of requirement or dependency for the usage of this module.
+ * components in a solution
+ * external dependencies
+ * other modules
+-->
+
+## Usage
+<!--
+ Provide an example configuration of the Module in a Solution.
+
+ Ensure all parameters are listed, with "/* optional */" where applicable
+-->
+```json
```

Comment:
I think we probably need to give a name to these kinds of plugins. They aren't specifically just modules they also include extensions and could potentially offer anything.

Reply:
Do you mean like "external plugins" - something to indicate that they should be reviewed before using?

If we have Extensions and Modules, and this is just something that implements both - so long as each of them are documented, then do we need a 3rd kind?